src/modules/service-catalog/components/service-catalog-list-item/LoadingState.tsx

```diff
@@ -0,0 +1,64 @@
+import { Col, Grid, Row } from "@zendeskgarden/react-grid";
+import { Skeleton } from "@zendeskgarden/react-loaders";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+import { MD, XXXL } from "@zendeskgarden/react-typography";
+import styled from "styled-components";
+
+const SkeletonItem = styled(Col)`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.space.sm};
+  height: 100%;
+  border-radius: ${(props) => props.theme.borderRadii.md};
+  padding: ${(props) => props.theme.space.md};
+  border: ${(props) => props.theme.borders.sm}
+    ${(props) => getColorV8("grey", 300, props.theme)};
+`;
+
+const TextSkeleton = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.space.xxs};
+  margin-bottom: ${(props) => props.theme.space.md};
+`;
+
+const StyledGrid = styled(Grid)`
+  padding: 0;
+`;
+
+const DescriptionSkeleton = styled.div`
+  font-size: ${(props) => props.theme.space.xs};
+`;
+
+const SkeletonCol = () => (
+  <Col xs={12} sm={6} md={4} lg={3}>
+    <SkeletonItem>
+      <XXXL>
+        <Skeleton width="40px" />
+      </XXXL>
+      <TextSkeleton>
+        <MD>
+          <Skeleton width="75%" />
+        </MD>
+        <DescriptionSkeleton>
+          <Skeleton width="90%" />
+        </DescriptionSkeleton>
+        <DescriptionSkeleton>
+          <Skeleton width="50%" />
+        </DescriptionSkeleton>
```

Comment:
Why do we need these `DescriptionSkeleton` wrappers? We can just wrap the two skeletons with `SM`

Reply:
`SM` was too big, at the design there is 8px and `SM` has 12px.

Reply:
But I changed it as you suggested, maybe it doesn't matter that much.

Reply:
I don't think it matters that much and probably it is better to just use what Garden provides by default.